src/components/MainPage.js

```diff
@@ -0,0 +1,188 @@
+import CameraIcon from "@mui/icons-material/PhotoCamera";
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+import CssBaseline from "@mui/material/CssBaseline";
+import Grid from "@mui/material/Grid";
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import Container from "@mui/material/Container";
+import { createTheme, ThemeProvider } from "@mui/material/styles";
+import Labels from "./Labels";
+import { Chip } from "@mui/material";
+import Progress from "./Progress";
+import reactJsLogo from "../logoImages/react-logo.png";
+import jsLogo from "../logoImages/js.svg";
+import dataStructureLogo from "../logoImages/data-structure.png";
+
+const quizCards = [
+  {
+    id: "react-basic",
+    title: "React Basics",
+    category: "react",
+    level: "basic",
+    labels: ["frontend"],
+    image: reactJsLogo,
+  },
+  {
+    id: "react-middle",
+    title: "React Mid-Level",
+    category: "react",
+    level: "mid-level",
+    labels: ["frontend"],
+    image: reactJsLogo,
+  },
+  {
+    id: "js-basic",
+    title: "JS Basics",
+    category: "javaScript",
+    level: "basic",
+    labels: ["frontend", "backend"],
+    image: jsLogo,
+  },
+  {
+    id: "js-functions",
+    title: "JS Functions",
+    category: "javaScript",
+    level: "basic",
+    labels: ["frontend", "backend"],
+    image: jsLogo,
+  },
+  {
+    id: "js-middle",
+    title: "JS Mid-Level",
+    category: "javaScript",
+    level: "mid-level",
+    labels: ["frontend", "backend"],
+    image: jsLogo,
+  },
+  {
+    id: "data-structures",
+    title: "Data structures",
+    category: "data structures",
+    level: "middle",
+    labels: ["frontend", "backend"],
+    image: dataStructureLogo,
+  },
+];
+const userProgress = {
+  ["userId"]: [
+    {
+      id: "react-basic",
+      attemptsNumber: "",
+      progress: 50,
+    },
+    {
+      id: "react-middle",
+      attemptsNumber: "",
+      progress: 10,
+    },
+    {
+      id: "js-basic",
+      attemptsNumber: "",
+      progress: 80,
+    },
+    {
+      id: "js-functions",
+      attemptsNumber: "",
+      progress: 100,
+    },
+  ],
+};
+
+const defaultTheme = createTheme();
+
+export default function MainPage() {
+  return (
+    <ThemeProvider theme={defaultTheme}>
+      <CssBaseline />
+      <main>
+        <Box
+          sx={{
+            pt: 6,
+          }}
+        >
+          <Container maxWidth="sm">
+            <Typography
+              component="h1"
+              variant="h4"
+              align="center"
+              color="text.primary"
+              gutterBottom
+            >
+              CHOOSE A QUIZ
+            </Typography>
+          </Container>
+        </Box>
+        <Container sx={{ py: 8 }} maxWidth="md">
+          <Grid container spacing={4}>
```

Comment:
This is looking great Eva! 

It'd be good to make the action button, the Start Quiz button, the same height, width, and location in each card so that the user has a consistent place to find the action on the card.

![image](https://github.com/Code-the-Dream-School/ee-prac-team3-front/assets/6400323/9b8464ab-c566-47e3-92ea-6bbe71f2d7ac)